build/BUILD.bazel

```
@@ -47,6 +48,18 @@ install_webui_cmd = select({
     "@kong//:skip_webui_flags": "\n",
 })
 
+install_wasm_filters_cmd = select({
+    "@kong//:wasmx_flag": """
+            mkdir -pv ${BUILD_DESTDIR}/kong/include/filters
+        """ + " ".join(["""
+            tpath="$(locations %s)"
+            fname="$(echo "${tpath}" | cut -d'/' -f2)"
+
+            cp -v "$tpath" "${BUILD_DESTDIR}/kong/include/filters/${fname}"
```

Can we swap this to `<dest>/kong/filters`?

```suggestion
            mkdir -pv ${BUILD_DESTDIR}/kong/filters
        """ + " ".join(["""
            tpath="$(locations %s)"
            fname="$(echo "${tpath}" | cut -d'/' -f2)"

            cp -v "$tpath" "${BUILD_DESTDIR}/kong/filters/${fname}"
```